core/src/main/java/com/google/instrumentation/trace/PropagationUtil.java

```diff
@@ -30,139 +28,76 @@
  * <li>version_id: 1-byte representing the version id.
  * <li>For version_id = 0:
  *     <ul>
- *     <li>version_format: &lt;trace-id&gt;&lt;span-id&gt;&lt;trace-options&gt;
- *     <li>trace-id: 16-byte array representing the trace_id.
- *     <li>span-id: 8-byte array representing the span_id.
- *     <li>trace-options: 4-byte array representing the trace_options. It is in little-endian order,
- *         if represented as an int.
+ *     <li>version_format: &lt;field&gt;&lt;field&gt;...
```

Comment:
I'm not sure.  I thought it was something like BNF at first, but then I didn't understand "`TraceId: (field_id = 0, len = 16, default = "0000000000000000")`".  Maybe it would help if you emphasized that TraceId, SpanId, and TraceOptions are the fields described in the first line, for example:

```
...
* field_format: <field_id><field_format>
* Fields:
  0. TraceId: 16-byte array representing the trace_id.  Default = "0000000000000000"
  1. SpanId: 8-byte array representing the span_id.  Default = "00000000"
  2. TraceOptions: 1-byte array representing the trace_options. Default = "0"
* Fields MUST be encoded using the field id order (smaller to higher).
...
```